_data/simple-icons.json

```
@@ -7601,6 +7601,11 @@
             "hex": "FFFFFF",
             "source": "https://www.personio.com/"
         },
+        {
+            "title": "Pets at Home",
+            "hex": "4BA840",
+            "source": "https://media.petsathome.com/wcsstore/pah-as01/images/redesign/logo.svg"
```

```suggestion
            "source": "https://petsathome.com"
```

can you change this please?